spec/actions_specs/crashlytics_spec.rb

```diff
@@ -120,13 +120,13 @@
               end").runner.execute(:test)
 
             expect(command).to eq([
-              @crashlytics_bundle,
-              "wadus",
-              "secret",
-              "-ipaPath './fastlane/spec/fixtures/fastfiles/Fastfile1'",
-              "-notifications YES",
-              "-debug NO"
-            ])
+                                    @crashlytics_bundle,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.